ucc/aqc/mps_sequential.py

```diff
@@ -0,0 +1,326 @@
+import numpy as np
+from numpy.typing import NDArray
+import quimb.tensor as qtn  # type: ignore
+from qiskit import QuantumCircuit  # type: ignore
+from qiskit.quantum_info import Statevector  # type: ignore
+from .utils import calculate_entanglement_entropy_slope
+import warnings
+
+
+def gram_schmidt(matrix: NDArray[np.complex128]) -> NDArray[np.complex128]:
```

Comment:
(asking to understand) is there a reason to implement this yourself versus using scipy.linalag.qr and taking the columns in Q?